shared/devices/add-device/container.js

```diff
@@ -21,9 +21,15 @@ const mapDispatchToProps = (dispatch, {navigateUp}) => ({
   onCancel: () => dispatch(navigateUp()),
 })
 
-export default Container.namedConnect<RouteProps<{}, {}>, _, _, _, _>(
+export default Container.namedConnect<
+  RouteProps<{highlight: Array<'computer' | 'phone' | 'paper key'>}, {}>,
```

Comment:
does this need to be an array? don't we just highlight at most one thing?

Reply:
I saw some designs for a nudge to 'add another computer / phone', I thought it'd be nice to support highlighting both options if we want to.